tachyon/math/finite_fields/generator/generator.cc

```diff
@@ -123,6 +123,8 @@ int GenerationConfig::GenerateConfigHdr() const {
       "",
       "  constexpr static uint64_t kExtensionDegree = 1;",
       "",
+      "  constexpr static BigInt<%{n}> kCharacteristic = kModulus;",
```

Comment:
I'm not sure if we really need to add this field.

Reply:
After discussion, we've decided to call a Characteristic function from prime_field_base.h, to return value of Config::kModulus.